pkg/apis/eventing/v1beta2/eventtype_lifecycle.go

```diff
@@ -20,15 +20,18 @@ import (
 	corev1 "k8s.io/api/core/v1"
 
 	eventingv1 "knative.dev/eventing/pkg/apis/eventing/v1"
+	messagingv1 "knative.dev/eventing/pkg/apis/messaging/v1"
 	"knative.dev/pkg/apis"
 )
 
-var eventTypeCondSet = apis.NewLivingConditionSet(EventTypeConditionBrokerExists, EventTypeConditionBrokerReady)
+var eventTypeCondSet = apis.NewLivingConditionSet(EventTypeConditionBrokerExists, EventTypeConditionBrokerReady, EventTypeConditionChannelExists, EventTypeConditionChannelReady)
 
 const (
-	EventTypeConditionReady                           = apis.ConditionReady
-	EventTypeConditionBrokerExists apis.ConditionType = "BrokerExists"
-	EventTypeConditionBrokerReady  apis.ConditionType = "BrokerReady"
+	EventTypeConditionReady                            = apis.ConditionReady
+	EventTypeConditionBrokerExists  apis.ConditionType = "BrokerExists"
+	EventTypeConditionBrokerReady   apis.ConditionType = "BrokerReady"
+	EventTypeConditionChannelExists apis.ConditionType = "ChannelExists"
+	EventTypeConditionChannelReady  apis.ConditionType = "ChannelReady"
```

Comment:
I have a PR that basically focus on: if a `reference` exists.

See -> #7010 
(it still is named for  `Broker`, but that changes)